spec/components/rails_bootstrap/badge_component_spec.rb

```diff
@@ -0,0 +1,52 @@
+require "rails_helper"
+
+RSpec.describe RailsBootstrap::BadgeComponent, type: :component do 
+  context "variation attribute tests" do
+    it "renders default variation" do 
+      subject = render_inline(RailsBootstrap::BadgeComponent.new()) { "This is a default badge" }
+
+      expect(subject.css("span[class='badge badge-primary']").to_html).not_to be_empty
+    end
+
+    it "renders secondary style" do 
```

Comment:
May be name this test to "sets variation to secondary" or something more descriptive?